package.json

```diff
@@ -1,48 +1,61 @@
 {
   "name": "formsy-react",
   "version": "1.0.0",
-  "description": "A form input builder and validator for React JS",
+  "description": "A form input builder and validator for React",
+  "keywords": [
+    "form",
+    "forms",
+    "formsy",
+    "react",
+    "react-component",
+    "validation"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/christianalfoni/formsy-react",
+  "bugs": "https://github.com/christianalfoni/formsy-react/issues",
   "repository": {
     "type": "git",
     "url": "https://github.com/christianalfoni/formsy-react.git"
   },
+  "author": "Christian Alfoni",
+  "files": [
+    "lib"
+  ],
   "main": "lib/index.js",
   "scripts": {
-    "build": "NODE_ENV=production webpack -p --config webpack.production.config.js",
-    "test": "babel-node testrunner",
+    "build": "NODE_ENV=production webpack -p --config webpack.config.js && yarn run prepublish",
     "examples": "webpack-dev-server --config examples/webpack.config.js --content-base examples",
-    "prepublish": "rm -Rf ./lib && babel ./src/ -d ./lib/"
+    "lint": "eslint src/**/*.js",
+    "prepublish": "rm -Rf ./lib && babel ./src/ -d ./lib/",
+    "test": "babel-node testrunner"
   },
-  "author": "Christian Alfoni",
-  "license": "MIT",
-  "keywords": [
-    "react",
-    "form",
-    "forms",
-    "validation",
-    "react-component"
-  ],
   "dependencies": {
     "form-data-to-object": "^0.2.0",
-    "hoist-non-react-statics": "^2.2.1",
-    "react": "^15.0.0"
+    "prop-types": "^15.5.10"
   },
   "devDependencies": {
-    "babel-cli": "^6.6.5",
-    "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-2": "^6.5.0",
-    "create-react-class": "^15.6.0",
-    "jsdom": "^6.5.1",
-    "nodeunit": "^0.9.1",
-    "prop-types": "^15.5.10",
-    "react": "^15.0.0",
-    "react-addons-pure-render-mixin": "^15.0.0",
-    "react-addons-test-utils": "^15.0.0",
-    "react-dom": "^15.0.0",
-    "sinon": "^1.17.3",
-    "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
+    "babel-cli": "^6.24.1",
+    "babel-loader": "^7.1.1",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "eslint": "^3.19.0 || ^4.3.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.1.0",
+    "jsdom": "^11.1.0",
+    "nodeunit": "^0.11.1",
+    "react": "^15.6.1",
+    "react-addons-pure-render-mixin": "^15.6.0",
+    "react-addons-test-utils": "^15.6.0",
+    "react-dom": "^15.6.1",
+    "sinon": "^3.2.0",
+    "webpack": "^3.5.4",
+    "webpack-dev-server": "^2.7.1"
+  },
+  "peerDependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
```

Comment:
You're right! I've made a habit of using dependencies + devDependencies, ditching peer deps altogether 🙃, but this is the better route.